bundles/net.tourbook.web/WebContent-dev/tourbook/search/nls/messages.properties

```diff
@@ -22,7 +22,7 @@ Search_Options_Checkbox_EaseSearching_Tooltip             = <div class="tooltip-
                                                             visible.<br><br>\n\
                                                             \n\
                                                             <b>Extended Search</b><br>\n\
-                                                            It can be helpfull to deactivate <i>Use ease searching</i> when doing a search with Lucene query parser options.\n\
+                                                            It can be helpful to deactivate <i>Use ease searching</i> when doing a search with Lucene query parser options.\n\
```

Comment:
When UI texts in `net.tourbook.web` are modified, then the dojo files needs to be updated

![image](https://github.com/mytourbook/mytourbook/assets/1283445/eea3cf3d-1797-4e3d-92e9-fc17fb230e5f)

Reply:
I saw this but didn't understand where to put this: as a windows environment variable ? somewhere in a Java file ?
` "IS_DEBUG = false"`

Reply:
>I saw this but didn't understand where to put this: as a windows environment variable ? somewhere in a Java file ?
 "IS_DEBUG = false"


![image](https://github.com/mytourbook/mytourbook/assets/1283445/ea8c5ef3-2149-400a-a1f1-bf3ba5e5ca03)



![image](https://github.com/mytourbook/mytourbook/assets/1283445/b191da57-3304-4452-9205-21d4322006f1)

Reply:
You need the dojo toolkit https://github.com/mytourbook/mytourbook-dojo-toolkit which is configured in

![image](https://github.com/mytourbook/mytourbook/assets/1283445/7ff9e240-b707-415d-a032-fd08b76008a3)


This is the log when it runs successfully [CREATE-Dojo-Bundle.log](https://github.com/user-attachments/files/15709964/CREATE-Dojo-Bundle.log)

Reply:
> You need the dojo toolkit https://github.com/mytourbook/mytourbook-dojo-toolkit which is configured in
> 

I already have the toolkit on my machine

![image](https://github.com/mytourbook/mytourbook/assets/25235665/3339e153-9f2c-4dcd-8074-faba488e11c1)

Reply:
After restarting my machine and Eclipse, I was able to run the Ant script successfully
```

Buildfile: C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\build-files\CREATE-Dojo-Bundle.xml
cleanBuildFiles:
     [echo] Cleaning tourbook in Dojo Toolkit build folder
   [delete] Deleting directory C:\DAT\MT\mytourbook-dojo-toolkit\tourbook
cleanMTRelease:
     [echo] Cleaning MT release folder
   [delete] Deleting directory C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel
shrink:
     [echo] ### Copying src files into dojo root ...
     [copy] Copying 27 files to C:\DAT\MT\mytourbook-dojo-toolkit\tourbook
     [echo] Running dojo custom build...
     [java] processing profile resource C:/Users/frederic/git/MT/mytourbook/bundles/net.tourbook.web/build-files/mytourbook-app.profile.js
     [java] info(107) Package Version: package: dgrid; version: 0.4.0
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/dgrid/package.js
     [java] info(107) Package Version: package: dijit; version: 1.10.2
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/dijit/dijit.profile.js
     [java] info(107) Package Version: package: xstyle; version: 0.3.1
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/xstyle/package.js
     [java] info(107) Package Version: package: tourbook; version: 15.3.0
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/tourbook/build-profile.js
     [java] info(107) Package Version: package: dojo; version: 1.10.2
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/dojo/dojo.profile.js
     [java] info(107) Package Version: package: dstore; version: 1.0.0
     [java] processing profile resource C:/Users/frederic/git/MT/mytourbook/bundles/net.tourbook.web/build-files/mytourbook-app.profile.js
     [java] info(107) Package Version: package: dgrid; version: 0.4.0
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/dgrid/package.js
     [java] info(107) Package Version: package: dijit; version: 1.10.2
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/dijit/dijit.profile.js
     [java] info(107) Package Version: package: xstyle; version: 0.3.1
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/xstyle/package.js
     [java] info(107) Package Version: package: tourbook; version: 15.3.0
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/tourbook/build-profile.js
     [java] info(107) Package Version: package: dojo; version: 1.10.2
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/dojo/dojo.profile.js
     [java] info(107) Package Version: package: dstore; version: 1.0.0
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/dstore/package.js
     [java] info(107) Package Version: package: rql; version: 0.3.2
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/rql/package.js
     [java] info(107) Package Version: package: put-selector; version: 0.3.6
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/put-selector/package.js
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/dstore/package.js
     [java] info(107) Package Version: package: rql; version: 0.3.2
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/rql/package.js
     [java] info(107) Package Version: package: put-selector; version: 0.3.6
     [java] processing profile resource C:/DAT/MT/mytourbook-dojo-toolkit/put-selector/package.js
     [java] discovering resources...
     [java] starting reading resources...
     [java] starting processing raw resource content...
     [java] starting tokenizing resource...
     [java] starting processing resource tokens...
     [java] starting parsing resource...
     [java] starting processing resource AST...
     [java] warn(216) dojo/has plugin resource could not be resolved during build-time. plugin resource id: touch?../util/touch; reference module id: dgrid/extensions/DnD
     [java] warn(216) dojo/has plugin resource could not be resolved during build-time. plugin resource id: touch?./_DnD-touch-autoscroll; reference module id: dgrid/extensions/DnD
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/extensions/DnD; plugin: xstyle/css
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/List; plugin: xstyle/css
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/extensions/CompoundColumns; plugin: xstyle/css
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/extensions/ColumnHider; plugin: xstyle/css
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/extensions/ColumnResizer; plugin: xstyle/css
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/TouchScroll; plugin: xstyle/css
     [java] warn(216) dojo/has plugin resource could not be resolved during build-time. plugin resource id: touch?./util/touch; reference module id: dgrid/Selection
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/ColumnSet; plugin: xstyle/css
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/extensions/Pagination; plugin: xstyle/css
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dgrid/extensions/ColumnReorder; plugin: xstyle/css
     [java] warn(216) dojo/has plugin resource could not be resolved during build-time. plugin resource id: touch?./util/touch; reference module id: dgrid/Tree
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dijit/form/_RadioButtonMixin; plugin: dojo/query
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dijit/Fieldset; plugin: dojo/query
     [java] warn(216) dojo/has plugin resource could not be resolved during build-time. plugin resource id: dojo-bidi?./_BidiMixin; reference module id: dijit/_WidgetBase
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dijit/Tree; plugin: dojo/query
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dijit/RadioMenuItem; plugin: dojo/query
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dojo/request/node; plugin: dojo/node
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dojo/request/node; plugin: dojo/node
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dojo/request/node; plugin: dojo/node
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dojo/request/node; plugin: dojo/node
     [java] warn(216) dojo/has plugin resource could not be resolved during build-time. plugin resource id: dojo-firebug?./_firebug/firebug; reference module id: dojo/main
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dojo/request; plugin: dojo/request/default
     [java] warn(216) dojo/has plugin resource could not be resolved during build-time. plugin resource id: dom-addeventlistener?:./aspect; reference module id: dojo/on
     [java] warn(216) dojo/has plugin resource could not be resolved during build-time. plugin resource id: host-browser?dom-addeventlistener?:../on:; reference module id: dojo/request/watch
     [java] warn(224) A plugin dependency was encountered but there was no build-time plugin resolver. module: dojo/request/registry; plugin: dojo/request/default
     [java] starting executing global optimizations...
     [java] starting writing resources...
     [java] List.js.uncompressed.js:682: WARNING - unreachable code
     [java] 			if (! 1  && typeof this[fn + 'Attr'] === 'function') {
     [java] 			                                                     ^
     [java] List.js.uncompressed.js:723: WARNING - unreachable code
     [java] 					if (! 1  && typeof this[fn + 'Attr'] === 'function') {
     [java] 					                                                     ^
     [java] 0 error(s), 2 warning(s)
     [java] InlineEditBox.js.uncompressed.js:538: WARNING - Suspicious code. This code lacks side-effects. Is there a bug?
     [java] 			if(!this.editing){
     [java] 			^
     [java] 0 error(s), 1 warning(s)
     [java] _DateTimeTextBox.js.uncompressed.js:17: WARNING - Suspicious code. This code lacks side-effects. Is there a bug?
     [java] 	new Date("X"); // workaround for #11279, new Date("") == NaN
     [java] 	^
     [java] 0 error(s), 1 warning(s)
     [java] base.js.uncompressed.js:322: WARNING - unreachable code
     [java] 				for(var i = 0; i < args.length; i++){ // TODO: merge possible promises
     [java] 				                                ^
     [java] 0 error(s), 1 warning(s)
     [java] load-imports.js.uncompressed.js:39: WARNING - unreachable code
     [java] 		if(false && !has('dom-deep-import')){
     [java] 		                                    ^
     [java] load-imports.js.uncompressed.js:251: WARNING - unreachable code
     [java] 	var progIds = ['Msxml2.XMLHTTP', 'Microsoft.XMLHTTP', 'Msxml2.XMLHTTP.4.0'];
     [java] 	^
     [java] 0 error(s), 2 warning(s)
     [java] widget.js.uncompressed.js:87: WARNING - The use of the with structure should be avoided.
     [java] 						with(scope){
     [java] 						^
     [java] 0 error(s), 1 warning(s)
     [java] css.js.uncompressed.js:42: WARNING - Suspicious code. This code lacks side-effects. Is there a bug?
     [java] 				if(xCss){
     [java] 				^
     [java] 0 error(s), 1 warning(s)
     [java] SearchApp.js.uncompressed.js:341: WARNING - unreachable code
     [java] 	}else{
     [java] 	     ^
     [java] SearchApp.js.uncompressed.js:1166: WARNING - unreachable code
     [java] 	}else if( 0 ){
     [java] 	      ^
     [java] SearchApp.js.uncompressed.js:1434: WARNING - Suspicious code. This code lacks side-effects. Is there a bug?
     [java] 	new Date("X"); // workaround for #11279, new Date("") == NaN
     [java] 	^
     [java] SearchApp.js.uncompressed.js:5704: WARNING - unreachable code
     [java] 			if (! 1  && typeof this[fn + 'Attr'] === 'function') {
     [java] 			                                                     ^
     [java] SearchApp.js.uncompressed.js:5745: WARNING - unreachable code
     [java] 					if (! 1  && typeof this[fn + 'Attr'] === 'function') {
     [java] 					                                                     ^
     [java] SearchApp.js.uncompressed.js:10591: WARNING - Suspicious code. This code lacks side-effects. Is there a bug?
     [java] 				if(xCss){
     [java] 				^
     [java] SearchApp.js.uncompressed.js:28206: WARNING - Parse error. Non-JSDoc comment has annotations. Did you mean to start it with '/**'?
     [java] ], /* @callback */function(
     [java] ^
     [java] 0 error(s), 7 warning(s)
     [java] default.js.uncompressed.js:11: WARNING - unreachable code
     [java] 	}else if( 0 ){
     [java] 	      ^
     [java] 0 error(s), 1 warning(s)
     [java] ready.js.uncompressed.js:137: WARNING - unreachable code
     [java] 	if( false  && dojo.config.parseOnLoad && !dojo.isAsync){
     [java] 	                                                       ^
     [java] 0 error(s), 1 warning(s)
     [java] dojo.js.uncompressed.js:148: WARNING - unreachable code
     [java] 	if( 0 ){
     [java] 	       ^
     [java] dojo.js.uncompressed.js:158: WARNING - unreachable code
     [java] 	if( 0 ){
     [java] 	       ^
     [java] dojo.js.uncompressed.js:219: WARNING - unreachable code
     [java] 	if( false ){
     [java] 	           ^
     [java] dojo.js.uncompressed.js:236: WARNING - unreachable code
     [java] 	if( false ){
     [java] 	           ^
     [java] dojo.js.uncompressed.js:574: WARNING - unreachable code
     [java] 					if( false  && p=="async"){
     [java] 					                         ^
     [java] dojo.js.uncompressed.js:695: WARNING - unreachable code
     [java] 				if( 0 ){
     [java] 				       ^
     [java] dojo.js.uncompressed.js:703: WARNING - unreachable code
     [java] 		if( false ){
     [java] 		           ^
     [java] dojo.js.uncompressed.js:730: WARNING - unreachable code
     [java] 	}else{
     [java] 	     ^
     [java] dojo.js.uncompressed.js:746: WARNING - unreachable code
     [java] 	if( 0 ){
     [java] 	       ^
     [java] dojo.js.uncompressed.js:759: WARNING - unreachable code
     [java] 				if( 0  && comboPending && !comboPendingTimer){
     [java] 				                                             ^
     [java] dojo.js.uncompressed.js:858: WARNING - unreachable code
     [java] 				if( 0 ){
     [java] 				       ^
     [java] dojo.js.uncompressed.js:863: WARNING - unreachable code
     [java] 				if( false ){
     [java] 				           ^
     [java] dojo.js.uncompressed.js:950: WARNING - unreachable code
     [java] 			if( false ){
     [java] 			           ^
     [java] dojo.js.uncompressed.js:1055: WARNING - unreachable code
     [java] 				if( false  && legacyMode == sync && !plugin.executed){
     [java] 				                                                     ^
     [java] dojo.js.uncompressed.js:1237: WARNING - unreachable code
     [java] 				if( false ){
     [java] 				           ^
     [java] dojo.js.uncompressed.js:1312: WARNING - unreachable code
     [java] 	if( 0 ){
     [java] 	       ^
     [java] dojo.js.uncompressed.js:1412: WARNING - unreachable code
     [java] 				if( 0 ){
     [java] 				       ^
     [java] dojo.js.uncompressed.js:1452: WARNING - unreachable code
     [java] 					if( false  && legacyMode){
     [java] 					                         ^
     [java] dojo.js.uncompressed.js:1468: WARNING - unreachable code
     [java] 				if( false  && legacyMode){
     [java] 				                         ^
     [java] dojo.js.uncompressed.js:1553: WARNING - unreachable code
     [java] 				if( 0  && module.plugin && module.plugin.isCombo){
     [java] 				                                                 ^
     [java] dojo.js.uncompressed.js:1589: WARNING - unreachable code
     [java] 				if( false  && legacyMode && !waiting[mid]){
     [java] 				                                          ^
     [java] dojo.js.uncompressed.js:1726: WARNING - unreachable code
     [java] 	if( false ){
     [java] 	           ^
     [java] dojo.js.uncompressed.js:1738: WARNING - unreachable code
     [java] 	if( false ){
     [java] 	           ^
     [java] dojo.js.uncompressed.js:1805: WARNING - unreachable code
     [java] 		if( 0  && args[1]===defaultDeps){
     [java] 		                                ^
     [java] dojo.js.uncompressed.js:1839: WARNING - unreachable code
     [java] 				if( 0  && !targetModule){
     [java] 				                        ^
     [java] dojo.js.uncompressed.js:1849: WARNING - unreachable code
     [java] 			if( 0  && isArray(targetModule)){
     [java] 			                                ^
     [java] dojo.js.uncompressed.js:1867: WARNING - unreachable code
     [java] 	if( 0 ){
     [java] 	       ^
     [java] dojo.js.uncompressed.js:1897: WARNING - unreachable code
     [java] 	if( false ){
     [java] 	           ^
     [java] dojo.js.uncompressed.js:1927: WARNING - unreachable code
     [java] 		if( false ){
     [java] 		           ^
     [java] dojo.js.uncompressed.js:1934: WARNING - unreachable code
     [java] 		if( 0 ){
     [java] 		       ^
     [java] dojo.js.uncompressed.js:1939: WARNING - unreachable code
     [java] 	if( 0  && req.combo && req.combo.plugins){
     [java] 	                                         ^
     [java] dojo.js.uncompressed.js:1953: WARNING - unreachable code
     [java] 	if(! 1 ){
     [java] 	        ^
     [java] dojo.js.uncompressed.js:3478: WARNING - unreachable code
     [java] 	if( 0 ){
     [java] 	       ^
     [java] dojo.js.uncompressed.js:5406: WARNING - unreachable code
     [java] 	if(! 1 ){
     [java] 	        ^
     [java] dojo.js.uncompressed.js:13267: WARNING - unreachable code
     [java] 	if(! 1 ){
     [java] 	        ^
     [java] dojo.js.uncompressed.js:14478: WARNING - unreachable code
     [java] 	}else{
     [java] 	     ^
     [java] dojo.js.uncompressed.js:14946: WARNING - unreachable code
     [java] 	if( false  && dojo.config.parseOnLoad && !dojo.isAsync){
     [java] 	                                                       ^
     [java] 0 error(s), 37 warning(s)
     [java] sniff.js.uncompressed.js:13: WARNING - unreachable code
     [java] 	if(! 1 ){
     [java] 	        ^
     [java] 0 error(s), 1 warning(s)
     [java] kernel.js.uncompressed.js:140: WARNING - unreachable code
     [java] 	if( 0 ){
     [java] 	       ^
     [java] 0 error(s), 1 warning(s)
     [java] firebug.js.uncompressed.js:768: WARNING - unreachable code
     [java] 		}catch(e){
     [java] ^
     [java] 0 error(s), 1 warning(s)
     [java] has.js.uncompressed.js:19: WARNING - unreachable code
     [java] 	if(! 1 ){
     [java] 	        ^
     [java] 0 error(s), 1 warning(s)
     [java] loader.js.uncompressed.js:10: WARNING - unreachable code
     [java] 	if (! 1 ){
     [java] 	         ^
     [java] 0 error(s), 1 warning(s)
     [java] text.js.uncompressed.js:10: WARNING - unreachable code
     [java] 	}else{
     [java] 	     ^
     [java] 0 error(s), 1 warning(s)
     [java] config.js.uncompressed.js:171: WARNING - unreachable code
     [java] 	}else{
     [java] 	     ^
     [java] 0 error(s), 1 warning(s)
     [java] parser.js.uncompressed.js:9: WARNING - Suspicious code. This code lacks side-effects. Is there a bug?
     [java] 	new Date("X"); // workaround for #11279, new Date("") == NaN
     [java] 	^
     [java] 0 error(s), 1 warning(s)
     [java] js-array.js.uncompressed.js:347: WARNING - Suspicious code. This code lacks side-effects. Is there a bug?
     [java] 		return resul;t
     [java] 		             ^
     [java] js-array.js.uncompressed.js:347: WARNING - unreachable code
     [java] 		return resul;t
     [java] 		             ^
     [java] 0 error(s), 2 warning(s)
     [java] query.js.uncompressed.js:151: WARNING - Suspicious code. This code lacks side-effects. Is there a bug?
     [java] 			term != null ? term : term = {};
     [java] 			               ^
     [java] 0 error(s), 1 warning(s)
     [java] starting cleaning up...
     [java] starting reporting...
     [java] Report written to C:/Users/frederic/git/mytourbook-BUILD-dojo/mt-dojo-release/build-report.txt
     [java] Process finished normally.
     [java] 	errors: 0
     [java] 	warnings: 27
     [java] 	build time: 48.103 seconds
copyResources:
     [echo] ### Copying resources without js, dev->rel ...
     [copy] Copying 12 files to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\package.json to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\package.json
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\resources\NothingYouCouldDo.ttf to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\resources\NothingYouCouldDo.ttf
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\resources\Nunito-Bold.ttf to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\resources\Nunito-Bold.ttf
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\resources\mytourbook-icon.svg to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\resources\mytourbook-icon.svg
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\resources\tour-blog.css to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\resources\tour-blog.css
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\resources\tour-import-css3.css to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\resources\tour-import-css3.css
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\resources\tour-import-log.css to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\resources\tour-import-log.css
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\resources\tour-import.css to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\resources\tour-import.css
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\search\DialogSearchOptions.html to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\DialogSearchOptions.html
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\search\search.css to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\search.css
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\search\search.mthtml to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\search.mthtml
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\tourbook\search\startup.css to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\startup.css
     [echo] ### Copying nls resources, build->rel ...
     [copy] Copying 8 files to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls
     [copy] Copying C:\Users\frederic\git\mytourbook-BUILD-dojo\mt-dojo-release\tourbook\search\nls\SearchApp_ROOT.js to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls\SearchApp_ROOT.js
     [copy] Copying C:\Users\frederic\git\mytourbook-BUILD-dojo\mt-dojo-release\tourbook\search\nls\SearchApp_cs.js to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls\SearchApp_cs.js
     [copy] Copying C:\Users\frederic\git\mytourbook-BUILD-dojo\mt-dojo-release\tourbook\search\nls\SearchApp_de.js to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls\SearchApp_de.js
     [copy] Copying C:\Users\frederic\git\mytourbook-BUILD-dojo\mt-dojo-release\tourbook\search\nls\SearchApp_en.js to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls\SearchApp_en.js
     [copy] Copying C:\Users\frederic\git\mytourbook-BUILD-dojo\mt-dojo-release\tourbook\search\nls\SearchApp_es.js to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls\SearchApp_es.js
     [copy] Copying C:\Users\frederic\git\mytourbook-BUILD-dojo\mt-dojo-release\tourbook\search\nls\SearchApp_fr.js to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls\SearchApp_fr.js
     [copy] Copying C:\Users\frederic\git\mytourbook-BUILD-dojo\mt-dojo-release\tourbook\search\nls\SearchApp_it.js to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls\SearchApp_it.js
     [copy] Copying C:\Users\frederic\git\mytourbook-BUILD-dojo\mt-dojo-release\tourbook\search\nls\SearchApp_nl.js to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\nls\SearchApp_nl.js
     [echo] ### Copying MT root resources, dev->rel ...
     [copy] Copying 1 file to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel
     [copy] Copying C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-dev\favicon.ico to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\favicon.ico
     [echo] ### Copying Dojo resources, toolkit->rel ...
     [copy] Copying 113 files to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dgrid\css\skins\images\row_back.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dgrid\css\skins\images\row_back.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\accordionItemActive.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\accordionItemActive.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\accordionItemHover.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\accordionItemHover.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\buttonActive.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\buttonActive.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\buttonDisabled.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\buttonDisabled.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\buttonEnabled.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\buttonEnabled.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\buttonHover.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\buttonHover.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\calendarDayLabel.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\calendarDayLabel.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\calendarMonthLabel.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\calendarMonthLabel.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\calendarYearLabel.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\calendarYearLabel.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\checkmark.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\checkmark.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\checkmark.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\checkmark.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\checkmarkNoBorder.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\checkmarkNoBorder.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\checkmarkNoBorder.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\checkmarkNoBorder.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\circleIcon.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\circleIcon.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\circleIcon.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\circleIcon.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\comboArrowDown.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\comboArrowDown.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\dijitProgressBarAnim.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\dijitProgressBarAnim.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\dndCopy.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\dndCopy.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\dndMove.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\dndMove.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\dndNoCopy.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\dndNoCopy.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\dndNoMove.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\dndNoMove.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\dojoTundraGradientBg.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\dojoTundraGradientBg.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\dojoTundraGradientBg.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\dojoTundraGradientBg.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\doubleArrowDown.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\doubleArrowDown.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\doubleArrowUp.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\doubleArrowUp.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\folderClosed.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\folderClosed.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\folderOpened.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\folderOpened.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\i.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\i.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\i_half.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\i_half.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\i_half_rtl.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\i_half_rtl.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\i_rtl.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\i_rtl.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\leaf.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\leaf.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\loading.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\loading.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\menu.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\menu.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\minusButton.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\minusButton.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\no.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\no.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\noX.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\noX.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\plusButton.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\plusButton.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\popupMenuBg.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\popupMenuBg.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\preciseSliderThumb.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\preciseSliderThumb.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\preciseSliderThumb.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\preciseSliderThumb.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\preciseSliderThumbFocus.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\preciseSliderThumbFocus.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\preciseSliderThumbFocus.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\preciseSliderThumbFocus.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-1.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-1.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-2.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-2.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-3.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-3.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-4.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-4.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-5.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-5.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-6.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-6.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-7.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-7.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-8.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-8.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim-9.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim-9.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarAnim.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarAnim.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarEmpty.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarEmpty.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\progressBarFull.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\progressBarFull.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\radioButtonActive.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\radioButtonActive.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\radioButtonActiveDisabled.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\radioButtonActiveDisabled.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\radioButtonActiveHover.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\radioButtonActiveHover.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\radioButtonDisabled.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\radioButtonDisabled.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\radioButtonEnabled.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\radioButtonEnabled.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\radioButtonHover.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\radioButtonHover.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderEmpty.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderEmpty.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderEmptyVertical.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderEmptyVertical.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderFull.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderFull.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderFullFocus.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderFullFocus.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderFullVertical.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderFullVertical.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderFullVerticalFocus.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderFullVerticalFocus.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderThumb.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderThumb.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderThumbFocus.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderThumbFocus.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\sliderThumbFocus.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\sliderThumbFocus.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\smallArrowDown.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\smallArrowDown.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\smallArrowUp.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\smallArrowUp.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\splitContainerSizerH-thumb.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\splitContainerSizerH-thumb.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\splitContainerSizerH.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\splitContainerSizerH.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\splitContainerSizerV-thumb.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\splitContainerSizerV-thumb.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\splitContainerSizerV.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\splitContainerSizerV.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\spriteArrows.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\spriteArrows.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\spriteArrows.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\spriteArrows.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\spriteRoundedIconsSmall.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\spriteRoundedIconsSmall.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\spriteRoundedIconsSmall.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\spriteRoundedIconsSmall.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabActive.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabActive.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabClose.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabClose.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabClose.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabClose.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabCloseHover.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabCloseHover.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabCloseHover.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabCloseHover.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabDisabled.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabDisabled.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabEnabled.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabEnabled.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabHover.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabHover.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tabHover.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tabHover.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\titleBar.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\titleBar.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\titleBarBg.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\titleBarBg.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tooltipConnectorDown.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tooltipConnectorDown.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tooltipConnectorDown.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tooltipConnectorDown.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tooltipConnectorLeft.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tooltipConnectorLeft.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tooltipConnectorLeft.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tooltipConnectorLeft.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tooltipConnectorRight.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tooltipConnectorRight.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tooltipConnectorRight.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tooltipConnectorRight.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tooltipConnectorUp.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tooltipConnectorUp.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\tooltipConnectorUp.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\tooltipConnectorUp.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeExpand_leaf.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeExpand_leaf.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeExpand_leaf_rtl.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeExpand_leaf_rtl.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeExpand_loading.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeExpand_loading.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeExpand_minus.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeExpand_minus.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeExpand_minus_rtl.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeExpand_minus_rtl.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeExpand_mius.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeExpand_mius.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeExpand_plus.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeExpand_plus.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeExpand_plus_rtl.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeExpand_plus_rtl.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\treeHover.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\treeHover.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\validationInputBg.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\validationInputBg.gif
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\validationInputBg.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\validationInputBg.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dijit\themes\tundra\images\warning.png to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dijit\themes\tundra\images\warning.png
     [copy] Copying C:\DAT\MT\mytourbook-dojo-toolkit\dojo\resources\blank.gif to C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dojo\resources\blank.gif
compress:
     [gzip] Building: C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\dojo\dojo.js.jgz
     [gzip] Building: C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\SearchApp.js.jgz
     [gzip] Building: C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook.web\WebContent-rel\tourbook\search\mt-dojo.css.jgz
all:
BUILD SUCCESSFUL
Total time: 53 seconds

```

Reply:
👍